CHANGELOG.md

```diff
@@ -2,6 +2,39 @@
 
 ## Unreleased
 
+**NOTICE:** This release contains a significant restructuring of this repository and the generated
+classes as we evolve it towards a first stable artifact (although there is no stable artifact as of
+this release). Please read the notes carefully and refer to the PRs and associated issues for more
+details. Additionally, the [README](README.md) contains useful information that should be reviewed.
+
+* BREAKING: Split out incubating artifact. This repo now
+  publishes `io.opentelemetry.semconv:opentelemetry-semconv:{{version}}` for semantic conventions
+  which are stable (the artifact itself is not yet stable but we aim to stabilize eventually),
+  and `io.opentelemetry.semconv:opentelemetry-semconv-incubating:{{version}}` for semantic
+  conventions which are incubating (experimental) (this artifact will always have the `-alpha`
+  designation). As a part of this restructuring, old attributes which do not exist
+  in [semantic-conventions](https://github.com/open-telemetry/semantic-conventions), but which we
+  had retained manually with deprecated annotations, have been removed.
+  ([#41](https://github.com/open-telemetry/semantic-conventions-java/pull/41))
```

Comment:
I think there's only a few of these?
```suggestion
  designation). As a part of this restructuring, a few old attributes which were removed from
  [semantic-conventions](https://github.com/open-telemetry/semantic-conventions) (without being deprecated there) have been removed.
  ([#41](https://github.com/open-telemetry/semantic-conventions-java/pull/41))
```

Reply:
Here's the full list: https://github.com/open-telemetry/semantic-conventions-java/blob/1f90bc5dc388fb6ee13e179534098bbd5fe63fab/buildscripts/templates/SemanticAttributes.java.j2#L170-L941

If you do a search for `@Deprecated`, you get 74 results. For an accurate answer, you'd need to do an analysis to subtract the attributes which still remain but in a different class. So its somewhere between 0 and 74, and I'd guess that its closer to 74 than 0 considering that we only recently starting keeping things around in semconv.